src/core/reducers/categories.js

```diff
@@ -96,6 +104,47 @@ export function createEmptyCategoryList(): EmptyCategoryListType {
   }, {});
 }
 
+export const getCategories = (
+  categoriesState: CategoryMapType,
+  appName: string,
+  addonType: string,
+): Array<mixed> | null => {
+  if (
+    categoriesState &&
+    categoriesState[appName] &&
+    categoriesState[appName][addonType]
+  ) {
+    return Object.values(categoriesState[appName][addonType]);
+  }
+  return null;
+};
+
+export function getCategoryNames(
+  categoriesState: CategoryMapType,
+  addonCategories: AddonCategories,
+  appName: string,
+  addonType: string,
+): Array<mixed> | null {
```

Comment:
As above, this shouldn't be `Array<mixed>`, it should be an Array of the type you are returning.

Reply:
It should be string but when I put string in here, I am getting a flow type error 
```
Cannot return `relatedCategories` because  mixed [1] is incompatible with  string [2] in array element.
```
For removing this I used mixed.

Reply:
As mentioned above, try changing it back to `string`, and push a new commit with all of the changes, and I can help you debug the Flow errors.

Reply:
Ohk!